Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ViewOpenReviewRequestsController.java

```diff
@@ -0,0 +1,27 @@
+package edu.asu.conceptpower.web;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import edu.asu.conceptpower.app.manager.RequestsManager;
+
+/**
+ * This endpoint renders the UI of the open review requests 
+ * 
+ * @author Keerthivasan Krishnamurthy
+ * 
+ */
+
+@Controller
+public class ViewOpenReviewRequestsController {
+	@Autowired
+    private RequestsManager requestsMgr;
+
+	@GetMapping(value="/auth/request/all/open")
+    public String viewOpenReviewRequests(ModelMap model) {
+        model.addAttribute("openRequests", requestsMgr.getAllOpenReviews());
+        return "layouts/concepts/openrequests";
```

Comment:
I'm confused, the story is about "list of all concepts with open review requests", but you're returning requests here. The list of concepts should pretty much look like the list of search results, which from looking at the template is not the case?

Reply:
Ahh yes, Julia. I'm actually querying ReviewRequest Table with ```ReviewStatus.OPENED```. The new page looks like this. We can view the concept details by clicking on the concept I'd (A modal will popup)

<img width="1439" alt="Screen Shot 2020-11-05 at 10 32 42 AM" src="https://user-images.githubusercontent.com/55424824/98261254-43c9a580-1f52-11eb-971a-82dcf2cd54cb.png">

Reply:
Let me know if this is ok. If not, I'll join and return the concept entry table and render the complete data.

Reply:
We need more info than that (pretty much what's in the search results). This doesn't tell the user anything about the concepts, which is the key point of interest here.